src/components/SortDropdown.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import FormControl from '@material-ui/core/FormControl';
+import InputBase from '@material-ui/core/InputBase';
+import NativeSelect from '@material-ui/core/NativeSelect';
+
+const BootstrapInput = withStyles((theme) => ({
```

Comment:
Is it necessary to bootstrap the input style, or can you just style it directly using makeStyles (which is also provided by material UI) ?

Reply:
Tried to style it using makeStyle but there is an unexpected bold underline at the bottom of the dropdown(This is the default style defined in NativeSelect component). 
**Style in className**
![image](https://user-images.githubusercontent.com/26618651/132972280-f9050753-50b3-4824-80ef-f514c706186d.png)
**Style in Input **
![image](https://user-images.githubusercontent.com/26618651/132972569-83213d6f-cb55-4f4a-9d70-b64b2dd946ed.png)

Therefore, we probably want to style in the input element.
Renamed the 'BootstrapInput' to 'DropdownInput' to avoid confusion. And changed the 'fontWeight' to 500.